pkg/testutils/mock_vmis.go

```diff
@@ -0,0 +1,24 @@
+package testutils
```

Comment:
I think you can put this function under test/utils.go

Reply:
Will do. But what is the difference between `test/utils.go` and `pkg/testutils` ?

Reply:
`pkg/testutils` is used by the unit tests inside the pkg folder and `test/utils.go` is used by the functional tests.